src/email_utils.py

```diff
@@ -21,6 +23,23 @@ def _attachment_inline(b64data, mime_type, filename, content_id):
     return _attachment(b64data, mime_type, filename, content_id, 'inline')
 
 def send_email(logger: Logger, subject: str, msg: str, recipient: str | None = None, attachment: str | None = None) -> bool:
+    if not recipient:
+        recipient = app.config["ADMIN_EMAIL"]
+    if attachment:
+        attachment = attachment
+    else:
+        attachment = None
+    if os.getenv("USE_SENDGRID") and os.getenv("USE_SENDGRID") == "true":
+        log_msg = f"This is for sendgrid"
+        logger.info(log_msg)
+        result = send_email_sendgrid(subject, msg, recipient, attachment)
+    else:
+        log_msg = f"This is for postfix"
+        logger.info(log_msg)
+        result = send_email_postfix(subject, msg, recipient, attachment)
+    return result
```

Comment:
```suggestion
    from_email = From(app.config["FROM_EMAIL"], app.config["FROM_NAME"])
    if app.config["USE_SENDGRID"]:
        return send_email_sendgrid(logger, from_email, subject, msg, recipient, attachment)
    return send_email_postfix(logger, from_email, subject, msg, recipient, attachment)
```

you forgot to pass the `logger` parameter which is probably the main reason why nothing was working

Reply:
the app.config["USE_SENDGRID"] appears to be coming back as False even though this is configured:
```
app.config["USE_SENDGRID"] = boolify(os.getenv("SENDGRID_API_KEY"))
```

so the above will always go to send_email_postfix. making a change in src/app_core.py to have the following instead:
```
if os.getenv("SENDGRID_API_KEY"):
    app.config["USE_SENDGRID"] = True
else:
    app.config["USE_SENDGRID"] = False
```

instead of:
```
app.config["USE_SENDGRID"] = boolify(os.getenv("SENDGRID_API_KEY"))
```